### ISSUE TITLE
panics if `before_help` is set

### ISSUE THREAD

**Issue by jyn514** [ConnorGray/clap-markdown#21]
i have a CLI tool that sets `before_help` to a non-empty string. when i try and use clap_markdown to generate markdown help, it panics:
```
thread 'main' panicked at /home/jyn/.local/lib/cargo/registry/src/index.crates.io-6f17d22bba15001f/clap-markdown-0.1.3/src/lib.rs:241:5:
assertion failed: command.get_before_help().is_none()
stack backtrace:
   3: clap_markdown::build_command_markdown
             at /home/jyn/.local/lib/cargo/registry/src/index.crates.io-6f17d22bba15001f/clap-markdown-0.1.3/src/lib.rs:241:5
   4: clap_markdown::build_command_markdown
             at /home/jyn/.local/lib/cargo/registry/src/index.crates.io-6f17d22bba15001f/clap-markdown-0.1.3/src/lib.rs:329:9
   5: clap_markdown::write_help_markdown
             at /home/jyn/.local/lib/cargo/registry/src/index.crates.io-6f17d22bba15001f/clap-markdown-0.1.3/src/lib.rs:89:5
   6: clap_markdown::help_markdown_command
             at /home/jyn/.local/lib/cargo/registry/src/index.crates.io-6f17d22bba15001f/clap-markdown-0.1.3/src/lib.rs:31:5
```
it would be nice for this to be supported so i don't have to work around it in my own code.

**Comment by jyn514** [comment]
looks like this would be fixed by https://github.com/ConnorGray/clap-markdown/pull/16

**Comment by ConnorGray** [comment]
This should be fixed in the latest [clap-markdown 0.1.4](https://docs.rs/clap-markdown/0.1.4/clap_markdown/index.html) release, just published 🚂

If you still encounter issues, please feel free to reopen or file a new issue.

Thank you for using clap-markdown! 🙂
